distribution/openrct2.d.ts

```diff
@@ -2537,6 +2543,26 @@ declare global {
     type Widget =
         ButtonWidget | CheckboxWidget | ColourPickerWidget | CustomWidget | DropdownWidget | GroupBoxWidget |
         LabelWidget | ListViewWidget | SpinnerWidget | TextBoxWidget | ViewportWidget;
+        
+    type IconName = "arrow_down" | "arrow_up" | "chat" | "cheats" | "copy" | "empty" | "eyedropper" |
+        "fast_forward" | "game_speed_indicator" | "game_speed_indicator_double" | "glassy_recolourable" |
+        "hide_full" | "hide_partial" | "hide_scenery" | "hide_supports" | "hide_vegetation" | "hide_vehicles" |
+        "large_scenery" | "legacy_paths" | "link_chain" | "logo" | "logo_text" | "map_east" |
+        "map_east_pressed" | "map_gen_land" | "map_gen_noise" | "map_gen_trees" | "map_north" |
+        "map_north_pressed" | "map_south" | "map_south_pressed" | "map_west" | "map_west_pressed" |
+        "mountain_tool_even" | "mountain_tool_odd" | "multiplayer" | "multiplayer_desync" | "multiplayer_sync" |
+        "multiplayer_toolbar" | "multiplayer_toolbar_pressed" | "mute" | "mute_pressed" | "news_messages" |
+        "normal_selection_6x6" | "paste" | "path_railings" | "path_surface" | "paths" | "placeholder" |
```

Comment:
path_surface > path_surfaces

Reply:
No, they are not. All the tab images for objects should use plural - if these two do not, then their file names should be changed (in another PR).